src/main/java/com/accesso/challengeladder/services/UserService.java

```diff
@@ -88,34 +88,15 @@ public boolean updateUser(User updatedUser)
         }
     }
 
-    public User createUser(String username, String email)
-    {
-        User user = new User();
-        try
-        {
-
-            user.setEmail(email);
-            // creates a new user in the DB
-            userDao.create(user);
-        }
-        catch (Exception e)
-        {
-            logger.error("Exception..." + e.getMessage());
-            return null;
-        }
-        return user;
-    }
-
     public User getUser(String userId) throws SQLException
     {
         User user = null;
         if (userId != null)
         {
             user = userDao.queryForId(userId);
+            getMatchRecord(user);
```

Comment:
I will personally rename the getMatchRecord method to reflect that is updating the User object (i.e.: updateUserWinLosses) or make it returns the object so it's implicit what it does.